configs/babylon-integration/consumer-fpd.conf

```diff
@@ -31,7 +31,7 @@ MaxSubmissionRetries = 100
 FastSyncInterval = 20s
 
 ; The maximum number of blocks to catch up for each fast sync
-FastSyncLimit = 10
+FastSyncLimit = 100
```

Comment:
any reason to change this? have we tested this number in a running FP yet?

Reply:
my worry is if this is too large, it might break. if we change it to a very large number 10000000000, it will likely break.

100 is probably fine but my point is for any changes, we should test it out if we can't be 100% sure if it won't break anything

if not, we shouldn't change it if it's untested

Reply:
changing it to make fast sync quicker. 

yes, i tested it on the first testnet we created in Singapore.

Reply:
got it. this still means we haven't tested it in this PR.

iirc we changed some logic in FP (related to fast sync?) since the Signapore trip, so ideally we should test it again b/c there is no guarantee this will work.

we should at least mention that this will be tested in a new deployment. just to note that this is sth we haven't tested it

Reply:
it's simialar to that we cannot use past test results to justify new code changes.

Reply:
no, i had added this changes into the PR https://github.com/Snapchain/babylon-deployment/pull/41, and tested it with the new deployment, devnet4

Reply:
ah ok. that's good to know